image_od.py

```diff
@@ -22,6 +22,89 @@
 import os.path
 import numpy as np
 
+default_mask_colors: dict[str, list[int]] = {
```

Comment:
If no mask_colors are provided in the input arguments, can we use random colors instead (using def get_rand_color())?

alternatively, we could store these constants in a configuration file such as utils/colors.json or utils/colors.txt for better management

Reply:
We were using random colors before but I feel like that caused an issue with consistency. So for example, a person would be highlighted red at one time and the next time suddenly change to yellow. 
Another reason for this list of colors is that this are the same colors from the gstreamer plugins. So there'll be some uniformity if someone decides to run inference frame by frame from a video and compare the results.

The colors.json is actually a great idea and I was thinking of doing that. I'll also update the code to fallback to random mask colors in case the model wasn't trained on COCO.